assets/src/edit-story/components/form/text.js

```diff
@@ -40,6 +41,12 @@ const StyledInput = styled(Input)`
   padding-left: ${({ label }) => (label ? 6 : 0)}px;
   letter-spacing: ${({ theme }) => theme.fonts.body2.letterSpacing};
   ${({ textCenter }) => textCenter && `text-align: center`};
+  ${({ color, theme }) =>
+    color &&
```

Comment:
Redundant check for `color` (just `color === 'white'`) should suffice?

How are the changes in this file related to the font picker preview though btw?

Reply:
Hm, probably due to the previous version of this PR having an input field to search. Will revert this part.

Reply:
Done in 53be2065.